Psychtoolbox/PsychDemos/DrawFormattedText2Demo.m

```diff
@@ -0,0 +1,138 @@
+% DrawFormattedText2Demo
+%
+% Draws formatted text, shows how to align and center text vertically
+% and/or horizontally, how line-breaks are introduced, how to transform
+% text.
+%
+% Press any key to cycle through different demo displays.
+%
+% see also: PsychDemos, Screen DrawText?, DrawSomeTextDemo, DrawFormattedText2Demo
+
+% 09/15/16    dcn    Wrote it.
+
+try
+    AssertOpenGL;   % check PTB is functioning
```

Comment:
maxval = Screen('ColorRange', windowPtr);
Atm. the text renderer plugin only resolves with 8 bpc, so that scaling would work. But it would be more future proof to allow passing 0-1 values directly if that's doable.